lib/cli/src/store.rs

```diff
@@ -19,16 +19,16 @@ pub struct StoreOptions {
     #[structopt(flatten)]
     compiler: CompilerOptions,
 
-    /// Use JIT Engine.
-    #[structopt(long, conflicts_with_all = &["native", "object_file"])]
+    /// Use the JIT Engine.
+    #[structopt(long, conflicts_with_all = &["shared_library", "object_file"])]
     jit: bool,
 
-    /// Use Native Engine.
+    /// Use the Shared Library Engine.
     #[structopt(long, conflicts_with_all = &["jit", "object_file"])]
-    native: bool,
+    shared_library: bool,
```

Comment:
Perhaps add an alias of `native`? Unsure what we want to do in regards to CLI flag stability

Reply:
The 1.0 is not released yet, not sure we want to have compatibility between alpha, beta, and stable version 🤔.